controllers/comments.controllers.js

```diff
@@ -0,0 +1,16 @@
+const { insertComment } = require("../models/comments.models");
+
+function postComment(request, response, next) {
+    const {article_id} = request.params
+    const {username, body} = request.body
+    if (!username || !body) {
+        return response.status(400).send({msg: 'bad request'})
+    }
+    const newComment = {...request.body, article_id}
```

Comment:
It would have made more sense to send over 3 parameters, so we could access them directly in the model from the parameters, rather than having to then destructure them

Reply:
For instance
insertComment(username, body, article_id)